DJHelper/DJHelper/EventPageViewController.swift

```diff
@@ -68,3 +122,27 @@ extension EventPageViewController: UITableViewDataSource, UITableViewDelegate {
         return UITableViewCell()
     }
 }
+
+extension EventPageViewController: MFMailComposeViewControllerDelegate {
+    func mailComposeController(_ controller: MFMailComposeViewController, didFinishWith result: MFMailComposeResult, error: Error?) {
+
+        guard error == nil else {
+            print("Mail finished with error: \(String(describing: error))")
+            return
+        }
+
+        switch result {
+
+        case .cancelled:
+            self.dismiss(animated: true, completion: nil)
+        case .saved:
+            return
+        case .sent:
+            self.dismiss(animated: true, completion: nil)
```

Comment:
What are your thoughts on us providing an alert here saying it was successfully sent and then dismissing the VC?